### DISCUSSION TITLE
arm64  execute : go run github.com/cilium/ebpf/cmd/bpf2go  , it will be blocked.  When kill it,  it only produces *_bpfel.o file,  and no *_bpfel.go, etc

### DISCUSSION THREAD

**Discussion by flytoofly** [cilium/ebpf#836] [Q&A]
arm64  execute : go run github.com/cilium/ebpf/cmd/bpf2go  , it will be block. it only produces *_bpfel.o file,  and no *_bpfel.go, etc

**Comment by flytoofly** [comment]
[analysis result]: it is  beause of  invoking  clang stuck,  but bpf2go should be killed when it is timeout.

**Reply by lmb** [reply]
If you think this is a bug then please create an issue and provide a viable reproducer. As it stands I can't really help you here.
